server/models/Collective.js

```diff
@@ -563,7 +564,9 @@ const Collective = sequelize.define(
     tags: {
       type: DataTypes.ARRAY(DataTypes.STRING),
       set(tags) {
-        this.setDataValue('tags', cleanTags(tags));
+        const sanitizedTags = sanitizeTags(tags);
+        validateTags(sanitizedTags);
+        this.setDataValue('tags', sanitizedTags);
```

Comment:
I suppose this is only triggered when creating or editing the field, which prevents validating it if the collective doesn't touch the field?

Reply:
@Betree Yes, only triggered when creating or editing the field.

However, worth knowing is that the Collective Info settings page does save all fields in the form, so it would be triggered there regardless of whether you changed the tags or not.